common/src/main/java/com/gregtechceu/gtceu/common/data/GTItems.java

```diff
@@ -707,7 +707,7 @@ public static ICustomDescriptionId cellName() {
             .onRegister(attach(new CoverPlaceBehavior(GTCovers.PUMPS[12])))
             .onRegister(attach(new TooltipBehavior(lines -> {
                 lines.add(Component.translatable("item.gtceu.electric.pump.tooltip"));
-                lines.add(Component.translatable("gtceu.universal.tooltip.fluid_transfer_rate", 1280 * 64 * 64 * 4 / 20));
+                lines.add(Component.translatable("gtceu.universal.tooltip.fluid_transfer_rate", 1280 * 64 * 64 * 64 * 64 / 20));
```

Comment:
I can't add an inline suggestion for some reason, but probably needs `1280L` here too